acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature

```diff
@@ -465,23 +465,22 @@ Feature: Open order with member tenant location and verify instance, holding, an
     And match physicalItemAfterOpenOrder2 != null
     And match physicalItemAfterOpenOrder2.status.name == 'Order closed'
 
-
-  Scenario: Verify existance of Inventory in member tenant after reopining order with deleteHoldings=true
```

Comment:
Seems diff showing confusing defference for this feature. Actually these scenarios duplicate and exissts in reopen-and-change-isntance connection feature. Removed duplicate features and added new test cases.
https://github.com/folio-org/folio-integration-tests/blob/7f79cbc6777b9736a72672f54164e31a48284cf8/acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature#L469

https://github.com/folio-org/folio-integration-tests/blob/7f79cbc6777b9736a72672f54164e31a48284cf8/acquisitions/src/main/resources/thunderjet/consortia/features/reopen-and-change-instance-connection-order-with-locations-from-different-tenants.feature#L34